spec/features/user_creates_a_published_moment_spec.rb

```diff
@@ -91,7 +91,7 @@
 
       find('#moment_comment_switch').click
       find('#moment_publishing_switch').click
-      find('#submit').click
+      find('#submit', wait: 10).click
```

Comment:
Is the `wait` necessary? I'd like to avoid using it unless absolutely needed.

Reply:
I'll take it out. It seemed like I got some failures before. let's see